scripts/dqlite/cmd/main.go

```diff
@@ -183,6 +200,42 @@ func main() {
 	<-ctx.Done()
 }
 
+func dumpDB(ctx context.Context, dial client.DialFunc, db *sql.DB, address, dir string) error {
+	cli, err := client.New(ctx, address, client.WithDialFunc(dial))
+	if err != nil {
+		return fmt.Errorf("client.New failed %w", err)
+	}
+
+	if dir == "" {
+		var err error
+		dir, err = os.Getwd()
+		if err != nil {
+			return fmt.Errorf("os.Getwd() failed %w", err)
+		}
+		dir = filepath.Join(dir, "dump")
+
+	}
+	if err := os.MkdirAll(dir, 0755); err != nil {
+		return fmt.Errorf("os.MkdirAll() failed %w", err)
+	}
+
+	files, err := cli.Dump(ctx, "tmp")
```

Comment:
This also seems to require the database name. I certainly would think that if we are connecting to the real database we would need "controller" or "model-$UUD" or something. Plausibly it should be passed in from the dbflag parameter.